docs/clients_openai.md

```diff
@@ -0,0 +1,95 @@
+# OpenAI API Clients {#ovms_docs_clients_openai}
+
+```{toctree}
+---
+maxdepth: 1
+hidden:
+---
+
+Chat API <ovms_docs_rest_api_chat>
+demo <https://github.com/openvinotoolkit/model_server/tree/main/demos/continuous_batching/>
+LLM calculator <ovms_docs_llm_caclulator>
+```
+## Introduction
+Beside Tensorflow Serving API and KServe API frontends, the model server has now option to delegate the REST input deserialization and output serialization to a MediaPipe graph. A custom calculator can implement any form of REST API including streaming based on [Server-sent events](https://html.spec.whatwg.org/multipage/server-sent-events.html#server-sent-events).
```

Comment:
I think this is way better server side events specification page:

https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events

https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events#examples